lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, config):
         self.ont_mappings = load_ontology_mappings('/kb/module/data')
         self.code_table = 11
         self.re_api_url = config.re_api_url
+        self.used_twice_identifiers = {}
```

Comment:
Can you add a quick comment here stating the purpose of this var

Reply:
added comment